reporter/src/main/kotlin/utils/SpdxDocumentModelMapper.kt

```diff
@@ -23,35 +23,205 @@ import java.time.Instant
 import java.util.UUID
 
 import org.ossreviewtoolkit.model.Environment
+import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.model.utils.PackageConfigurationProvider
+import org.ossreviewtoolkit.model.utils.getDetectedLicensesWithCopyrights
+import org.ossreviewtoolkit.spdx.SpdxConstants
+import org.ossreviewtoolkit.spdx.SpdxExpression
 import org.ossreviewtoolkit.spdx.SpdxLicense
 import org.ossreviewtoolkit.spdx.model.SpdxCreationInfo
 import org.ossreviewtoolkit.spdx.model.SpdxDocument
+import org.ossreviewtoolkit.spdx.model.SpdxPackage
+import org.ossreviewtoolkit.spdx.model.SpdxRelationship
 import org.ossreviewtoolkit.utils.ORT_FULL_NAME
+import org.ossreviewtoolkit.utils.ProcessedDeclaredLicense
+import org.ossreviewtoolkit.utils.stripCredentialsFromUrl
 
 /**
  * A class for mapping [OrtResult]s to [SpdxDocument]s.
  */
 object SpdxDocumentModelMapper {
-    fun map(@Suppress("UNUSED_PARAMETER") ortResult: OrtResult, params: SpdxDocumentParams): SpdxDocument {
-        val documentUuid = UUID.randomUUID()
+    data class SpdxDocumentParams(
+        val documentName: String,
+        val documentComment: String,
+        val creationInfoComment: String
+    )
+
+    fun map(
+        ortResult: OrtResult,
+        packageConfigurationProvider: PackageConfigurationProvider,
+        params: SpdxDocumentParams
+    ): SpdxDocument {
+        val spdxPackageIdGenerator = SpdxPackageIdGenerator()
+        val packages = mutableListOf<SpdxPackage>()
+        val relationships = mutableListOf<SpdxRelationship>()
+
+        val rootPackage = SpdxPackage(
+            spdxId = spdxPackageIdGenerator.nextId("root-package"),
+            copyrightText = SpdxConstants.NOASSERTION,
+            downloadLocation = SpdxConstants.NOASSERTION,
+            filesAnalyzed = false,
+            homepage = SpdxConstants.NOASSERTION,
+            licenseConcluded = SpdxConstants.NOASSERTION,
+            licenseDeclared = SpdxConstants.NOASSERTION,
+            name = "Root package",
+            summary = "An artificial stub package describing the root of the dependency tree."
```

Comment:
Summary is optional so I would omit it unless it can be specified by the user by passing a CLI param to this reporter. See also https://spdx.github.io/spdx-spec/3-package-information/#318-package-summary-description

Reply:
Done.